src/Symfony/Component/HttpKernel/DependencyInjection/AddCacheClearerPass.php

```diff
@@ -9,7 +9,7 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler;
+namespace Symfony\Component\HttpKernel\DependencyInjection;
```

Comment:
yo cannot just move a class. It is a BC break (as it deletes the old class). You need to keep the class in FrameworkBundle (making it extend the new one to avoid duplication) but deprecating it